setup.py

```diff
@@ -17,7 +17,7 @@
 # Give setuptools a hint to complain if it's too old a version
 # 30.3.0 allows us to put most metadata in setup.cfg
 # Should match pyproject.toml
-SETUP_REQUIRES = ['setuptools >= 30.3.0']
+SETUP_REQUIRES = ['setuptools >= 30.3.0', 'pynwb==1.0.3', 'hdmf==1.2.0']
```

Comment:
Let's not pin versions in setup.py - only minimally required version.
requirements.txt (if desired) or pipenv.lock is the way to go for pinning

Reply:
For pynwb we can specify `,!=1.1.0` (or something like that, I believe there are examples in nibabel) to avoid installation of the buggy version

Reply:
Ok yeah I agree open requirements are better, I just wanted to avoid the buggy versions right now.

Reply:
in 44b59755b9380776a357e8ab6cd503c4f3960d45 pushed all needed ;)